src/bgw/job.c

```diff
@@ -451,6 +453,24 @@ ts_bgw_job_permission_check(BgwJob *job)
 				(errcode(ERRCODE_INSUFFICIENT_PRIVILEGE),
 				 errmsg("insufficient permssions to alter job %d", job->fd.id)));
```

Comment:
```suggestion
				 errmsg("insufficient permissions to alter job %d", job->fd.id)));
```

Noticed a typo here. Realized not in your code, but seems worthwhile to fix (here or in other PR).